docs/content/v0.20/examples.md

```diff
@@ -58,7 +58,9 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
 6. Follow the steps to [create and register two clusters with OCM](example-wecs.md).
 
 
-7. (optional) Check all deployments and statefulsets running in the hosting cluster:
+7. (optional) Check all deployments and statefulsets running in the hosting cluster. Expect to 
+see the wds1 kubestellar-controller-manager created in the wds1-system namespace and the imbs1 
+statefulset created in the imbs1-system namespace.
```

Comment:
I am not sure that this is all the checking that @pdettori intended the reader to do, and I see that the e2e test is also doing more checking.

Here is what I see from a recent run of an e2e test:
```
+ :
+ : -------------------------------------------------------------------------
+ : Get all deployments and statefulsets running in the hosting cluster
+ :
+ kubectl --context kind-kubeflex get deployments,statefulsets --all-namespaces
NAMESPACE            NAME                                             READY   UP-TO-DATE   AVAILABLE   AGE
ingress-nginx        deployment.apps/ingress-nginx-controller         1/1     1            1           3m48s
kube-system          deployment.apps/coredns                          2/2     2            2           3m50s
kubeflex-system      deployment.apps/kubeflex-controller-manager      1/1     1            1           3m5s
local-path-storage   deployment.apps/local-path-provisioner           1/1     1            1           3m49s
wds1-system          deployment.apps/kube-apiserver                   1/1     1            1           111s
wds1-system          deployment.apps/kube-controller-manager          1/1     1            1           111s
wds1-system          deployment.apps/kubestellar-controller-manager   1/1     1            1           65s

NAMESPACE         NAME                                   READY   AGE
imbs1-system      statefulset.apps/vcluster              1/1     2m36s
kubeflex-system   statefulset.apps/postgres-postgresql   1/1     3m46s
```

I think @pdettori also intended the reader to check that the number of ready replicas (https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L498) equals the number of desired replicas (https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L490) for each Deployment.

I see that the e2e change below expects 4 hits rather than just 2.

Reply:
@MikeSpreitzer I think that the above is good enough.  I prefer not to explicitly list all the resulting objects and the exact expected  deployments. We will end up describing the exact environment after each step which we don't want to do. The more info on the expected output we add them more detailed validation of the textual README we need to do.
If this was a test that is focused on showing changes in deployments (e.g., increase/decrease  the number) than it may make sense to explicitly check the change happened.

Reply:
@MikeSpreitzer this was mainly to check that all deployments and stateful sets are in ready state. In particular, we care about 

```
ingress-nginx        deployment.apps/ingress-nginx-controller         1/1     1            1           3m48s
kubeflex-system      deployment.apps/kubeflex-controller-manager      1/1     1            1           3m5s
wds1-system          deployment.apps/kube-apiserver                   1/1     1            1           111s
wds1-system          deployment.apps/kube-controller-manager          1/1     1            1           111s
wds1-system          deployment.apps/kubestellar-controller-manager   1/1     1            1           65s

NAMESPACE         NAME                                   READY   AGE
imbs1-system      statefulset.apps/vcluster              1/1     2m36s
kubeflex-system   statefulset.apps/postgres-postgresql   1/1     3m46s
```

Reply:
I agree with @ezrasilvera, we should not fix unnecessary details in the documentation or the e2e test. I think there are two things to check here: (a) the existence of the Deployments and StatefulSets that @pdettori mentioned and (b) `Replicas == ReadyReplicas` for all Deployments and StatefulSets (since this example starts from a clean slate, any broken set indicates a problem), without explicitly listing them in the doc or test (people and code can be told to iterate over whatever table rows show up).

I think this PR is good enough to merge as is. Later I will suggest an improvement.

Reply:
Thanks @MikeSpreitzer